aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -404,6 +404,10 @@ module aptos_framework::fungible_asset {
     #[view]
     /// Return whether the provided address has a store initialized.
     public fun store_exists(store: address): bool {
+        store_exists_inline(store)
+    }
+
+    fun store_exists_inline(store: address): bool {
```

Comment:
Why this change?

Reply:
To save one function call? I'm not sure if it's worth making the code longer

Reply:
store_exists_inline is a single operation - so inling leaves the byte representation the same. If you prefer - I could just not use store_exists in this file and do "exists<>" itself throughout.